.circleci/config.yml

```diff
@@ -77,6 +77,9 @@ test_and_cover: &test_and_cover
 
       - run: composer update -n --prefer-dist
 
+      - run: |
+          [ -f /usr/local/etc/php/conf.d/xdebug.ini] || rm /usr/local/etc/php/conf.d/xdebug.ini
```

Comment:
I had to do this b/c I was getting a segfault on enabling the xdebug extension below. This disables the existing xdebug extension before downloading and enabling the latest xdebug from pecl below.

Reply:
Makes sense!

You can simplify this with `rm -f`:

```
❯ rm /tmp/asdf; echo $?
rm: /tmp/asdf: No such file or directory
1
❯ rm -f /tmp/asdf; echo $?
0
```